scripts/upgrade

```diff
@@ -226,7 +226,7 @@ chown -R $app: $final_path
 (cd "$final_path/src"
 ynh_exec_warn_less npm cache clean --force
 # Remove package-lock.json to prevent any shit during npm update...
-ynh_secure_remove --file="$final_path/src/package-lock.json"
+#ynh_secure_remove --file="$final_path/src/package-lock.json"
```

Comment:
You sure about that considering the comment just before ? (No idea what exactly that may refer to)

Reply:
OK, trying again with this line uncommented, but this was previously failing upgrade tests...